primeqa/mrc/README.md

```diff
@@ -249,13 +249,61 @@ Additionally, to specify a MRQA subset e.g. `SQuAD`, `NaturalQuestionsShort`, `T
        --dataset_filter_column_name subset
 ```
 
-Cross domain experiments can be run by running train and eval as separate processes. The
+Cross domain experiments can be run by running train and eval as separate processes.
 
  
 ### Custom Data
 
 Users can also train (fine-tune) and evaluate the MRC model on custom data by providing their own train_file and eval_file. Instructions for getting started are available [here](../../examples/custom_mrc/README.md).
 
+
+### Training with Multiple Datasets
+
+PrimeQA supports the training of MRC model with combination of multiple datasets, which are specified in "--train_fof" argument. Three formats, csv, jsonl and json, can be used to define "--train_file".
```

Comment:
Since the argument is `train_fof`, it is unclear what `train_file` here is, because it looks like another argument.

Reply:
Also, a brief but concrete description of the content of `train_fof` here (how many columns/fields and what they represent) would be useful.

Reply:
Can the argument for multiple datasets still be "--train_file" with a check to see if it's a file with a list of datasets?

Reply:
@ma-sultan Modified the README as suggested.
@seirasto The argument '--train_fof' was introduced by Arafat's distillation model training which has been merged into main branch. The distillation model training also uses multiple datasets, but uses them in a different way than this PR. I think it would be good to keep the convention that '--train_file' is for data file(s) with same format, and  '--train_fof' is for datasets or data files with different formats.